src/cli/commands/lint.ts

```diff
@@ -217,38 +214,40 @@ async function lint(name: string, flags: any, command: Lint, rules?: RuleCollect
   }
 }
 
-const skipRules = (rules: any, flags: any, command: Lint): any => {
+const skipRules = (rules: RuleCollection, flags: ILintConfig, command: Lint): RuleCollection => {
   const skippedRules: string[] = [];
   const invalidRules: string[] = [];
 
-  for (const rule of flags.skipRule) {
-    if (rule in rules) {
-      delete rules[rule];
-      skippedRules.push(rule);
-    } else {
-      invalidRules.push(rule);
+  if (flags.skipRule !== undefined) {
```

Comment:
Hmm, was this working before? Ah ok it was just passing because of the `any` as type, right? 🤦‍♂

Reply:
Yeah, we had `any` everywhere. I guess we normalize that data *somewhere*, but I wouldn't assume we do that for every scenario.